src/routes/+layout.svelte

```diff
@@ -109,3 +131,33 @@
   <!-- (pageFooter) -->
   <!-- (footer) -->
 </AppShell>
+
+<style>
+  nav {
+    display: flex;
+  }
+
+  ul {
+    display: flex;
+    margin: 0;
+    margin-left: auto;
+    list-style: none;
+  }
+
+  li {
+    margin-right: 20px;
+  }
+
+  nav ul li a {
+    display: inline-block;
+    padding: 10px 20px;
+    border-radius: 5px;
+    background-color: #08417a;
```

Comment:
Make sure to try and use the skeleton colors and fonts wherever possible.

Replace this with one of the variants or the predefined tailwind color classes.